src/steps/big-table/converters.ts

```diff
@@ -0,0 +1,222 @@
+import { bigtableadmin_v2 } from 'googleapis';
+import { createGoogleCloudIntegrationEntity } from '../../utils/entity';
+import { getGoogleCloudConsoleWebLink, getLastUrlPart } from '../../utils/url';
+import {
+  ENTITY_CLASS_BIG_TABLE_APP_PROFILE,
+  ENTITY_CLASS_BIG_TABLE_BACKUP,
+  ENTITY_CLASS_BIG_TABLE_CLUSTER,
+  ENTITY_CLASS_BIG_TABLE_INSTANCE,
+  ENTITY_CLASS_BIG_TABLE_OPERATION,
+  ENTITY_CLASS_BIG_TABLE_TABLE,
+  ENTITY_TYPE_BIG_TABLE_APP_PROFILE,
+  ENTITY_TYPE_BIG_TABLE_BACKUP,
+  ENTITY_TYPE_BIG_TABLE_CLUSTER,
+  ENTITY_TYPE_BIG_TABLE_INSTANCE,
+  ENTITY_TYPE_BIG_TABLE_OPERATION,
+  ENTITY_TYPE_BIG_TABLE_TABLE,
+} from './constants';
+
+export function getOperationKey(operation: bigtableadmin_v2.Schema$Operation) {
+  return `bigtable_operation:${operation.name}`;
+}
+
+export function getInstanceKey(instance: bigtableadmin_v2.Schema$Instance) {
+  return `bigtable_instance:${instance.name}`;
+}
+
+export function getAppProfileKey(
+  appProfile: bigtableadmin_v2.Schema$AppProfile,
+) {
+  return `bigtable_appProfile:${appProfile.name}`;
+}
+
+export function getClusterKey(cluster: bigtableadmin_v2.Schema$Cluster) {
+  return `bigtable_cluster:${cluster.name}`;
+}
+
+export function getBackupKey(backup: bigtableadmin_v2.Schema$Backup) {
+  return `bigtable_backup:${backup.name}`;
+}
+
+export function getTableKey(table: bigtableadmin_v2.Schema$Table) {
+  return `bigtable_table:${table.name}`;
+}
+
+export function getLocationKey(location: bigtableadmin_v2.Schema$Location) {
+  return `bigtable_location:${location.locationId}`;
+}
+
+export function createOperationEntity(
+  operation: bigtableadmin_v2.Schema$Operation,
+) {
+  return createGoogleCloudIntegrationEntity(operation, {
+    entityData: {
+      source: operation,
+      assign: {
+        _class: ENTITY_CLASS_BIG_TABLE_OPERATION,
+        _type: ENTITY_TYPE_BIG_TABLE_OPERATION,
+        _key: getOperationKey(operation),
+        name: operation.name,
+        done: operation.done,
+      },
+    },
+  });
+}
+
+export function createInstanceEntity({
+  instance,
+  projectId,
+}: {
+  instance: bigtableadmin_v2.Schema$Instance;
+  projectId: string | undefined | null;
+}) {
+  const instanceName = getLastUrlPart(instance.name!);
+
+  return createGoogleCloudIntegrationEntity(instance, {
+    entityData: {
+      source: instance,
+      assign: {
+        _class: ENTITY_CLASS_BIG_TABLE_INSTANCE,
+        _type: ENTITY_TYPE_BIG_TABLE_INSTANCE,
+        _key: getInstanceKey(instance),
+        name: instance.name,
+        displayName: instance.displayName!,
+        active: instance.state === 'READY',
```

Comment:
I've added this - we want to affect `active` field (in general) if we can based on some property (just point out an example)!